Automattic/wp-calypso
# Add PlanFeaturesList

Add `PlanFeaturesList` component which replaces `PlanList` if abtest `planFeatures` is activated. It's a part of plans redesign started in #5715.

Screenshots:

![selection_034](https://cloud.githubusercontent.com/assets/4988512/16206629/8defa67a-3729-11e6-83c8-5bdf25949b3b.jpg)

![selection_035](https://cloud.githubusercontent.com/assets/4988512/16206631/90d3a33c-3729-11e6-9403-bd0988742e15.jpg)

## Testing
1. Start Calypso with `ENABLE_FEATURES=manage/plan-features make run`
2. Select a site with Free plan;
3. Click on "Plans" in left sidebar;
4. You should see the redesigned plans depicted in the screenshots above.
5. Notice the URL — `/plans/features`.

Ensure that there are no functional changes to Calypso
1. Start Calypso with `make run`
2. You can update a wpcom plan normally
3. You can update a Jetpack plan normally

## Todo

- [x] Remove Compare plans from plan page and redirect `/plans/compare` to `/plans/features` (the new URL for plans redesign).
- [x] Add loading plan placeholders when we have no plans data.
- [x] Add fn to put items in cart (Upgrade btn currently doesn't work) — @artpi 
- [x] Remove abtest and add it in future PR

## Loading placeholders

This is how it looks with loading placeholders instead of plans:
![selection_050](https://cloud.githubusercontent.com/assets/4988512/16268817/22a2e964-3890-11e6-9a1d-3e195223b745.jpg)



cc @gwwar @rralian @mtias 


Test live: https://calypso.live/?branch=add/plan-features-list
